daemon/src/address_map.rs

```diff
@@ -61,6 +61,32 @@ where
             _ => Err(NotConnected::new::<A>()),
         }
     }
+
+    /// Sends a message to the actor stored with the given key.
+    pub async fn send_fallible<M>(
+        &self,
+        key: &K,
+        msg: M,
+    ) -> Result<anyhow::Result<()>, NotConnected>
```

Comment:
Do we need to qualify here? Don't think so :)

Reply:
We don't if we import `anyhow::Result`!